resources/views/PageDesign/Partials/Header/Navigation.twig

```diff
@@ -12,7 +12,13 @@
                         <img alt="{{ companyName }}" src="{{ companyLogo }}" class="img-fluid" />
                     </a>
                 {% endif %} 
-                <button v-open-mobile-navigation class="navbar-toggler d-lg-none p-3" type="button">
+                <button v-open-mobile-navigation
+                        class="navbar-toggler d-lg-none p-3"
+                        type="button"
```

Comment:
```suggestion
                        type="menu"
```

Wouldn't type="menu" be more accurate here?
https://developer.mozilla.org/en-US/docs/Web/API/HTMLButtonElement#htmlbuttonelement.type